src/Halogen/Aff/Driver/State.purs

```diff
@@ -160,7 +160,7 @@ initDriverState component input handler prjQuery lchs = do
       , pendingOuts
       , pendingHandlers
       , rendering: Nothing
-      , prjQuery
+      , prjQuery: prjQuery :: (forall a. f a -> Maybe (z a))
```

Comment:
Alternatively, I think you could annotate `ds`, which would probably a better option.